pkg/monitoring/metrics.go

```diff
@@ -40,6 +47,41 @@ func SetReadyGauge(isReady bool) {
 	}
 }
 
+func setKubemacpoolDeployedGauge(isDeployed bool) {
+	if isDeployed {
+		kmpDeployedGauge.Set(1)
+	} else {
+		kmpDeployedGauge.Set(0)
+	}
+}
+
+func ResetMonitoringComponents() {
```

Comment:
Are not these monitored components?

Reply:
They are! Thank you!!! 
finding the right wording brings me joy =)

Reply:
Done